src/main/java/org/snomed/snowstorm/syndication/services/importers/customversion/icd10/Icd10SyndicationService.java

```diff
@@ -0,0 +1,132 @@
+package org.snomed.snowstorm.syndication.services.importers.customversion.icd10;
+
+import org.snomed.snowstorm.core.data.services.ServiceException;
+import org.snomed.snowstorm.syndication.models.domain.SyndicationImportParams;
+import org.snomed.snowstorm.syndication.services.importers.SyndicationService;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Optional;
+
+import static java.util.Collections.singletonList;
+import static org.snomed.snowstorm.syndication.constants.SyndicationConstants.ICD10_TERMINOLOGY;
+import static org.snomed.snowstorm.syndication.constants.SyndicationConstants.LATEST_VERSION;
+import static org.snomed.snowstorm.syndication.constants.SyndicationConstants.LOCAL_VERSION;
+import static org.snomed.snowstorm.syndication.utils.CommandUtils.waitForProcessTermination;
+import static org.snomed.snowstorm.syndication.utils.FileUtils.findFile;
+
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+
+import java.util.TreeSet;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+@Service(ICD10_TERMINOLOGY)
+public class Icd10SyndicationService extends SyndicationService {
```

Comment:
Yes I've seen almost all of this before, so I don't know what exactly was not present in the other PRs, but I'll bring this one in now and see what we end up with - do a comparison with a local "pre-PR" branch...